src/app/game/buildings/building-details/building-detail.component.html

```diff
@@ -0,0 +1,38 @@
+<div id="overlay" >
+  <div id="modal-container">
+    <div class="building__details">
+      <img [src]="imgSrc" alt="">
+      <table class="building__details--table">
+        <thead>
+        <tr >
+          <td colspan="2">{{building.type}}  details</td>
```

Comment:
@HonzaDudek two spaces on purpose?

Reply:
Yes, to center the text above both columns.